src/SFA.DAS.Forecasting.Jobs.Application.UnitTests/Handlers/WhenHandlingEmployerLevyRefreshComplete.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerFinance.Messages.Events;
+using SFA.DAS.Forecasting.Domain.Services;
+using SFA.DAS.Forecasting.Jobs.Application.Triggers.Handlers;
+
+namespace SFA.DAS.Forecasting.Jobs.Application.UnitTests
+{
+    [TestFixture, Parallelizable]
+    public class WhenHandlingEmployerLevyRefreshComplete
+    {
+        private IFixture Fixture => new Fixture();
+        private Mock<ILevyForecastService> _levyForecastServiceMock;
+        private LevyCompleteTriggerHandler _sut;
+        private RefreshEmployerLevyDataCompletedEvent _event;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _levyForecastServiceMock = new Mock<ILevyForecastService>();
+            _sut = new LevyCompleteTriggerHandler(_levyForecastServiceMock.Object);
+
+            _event = Fixture
+                .Build<RefreshEmployerLevyDataCompletedEvent>()
+                .With(e => e.LevyImported, true)
+                .Create();
+        }
+
+        [Test]
+        [Category("UnitTest")]
+        public async Task If_No_Levy_Calculated_Should_Not_Trigger_Forecast()
+        {
+            // Arrange
+            _event.LevyImported = false;
+
+            // Act
+            await _sut.Handle(_event);
+
+            // Assert
+            _levyForecastServiceMock.Verify(mock => mock.TriggerLevyForecast(It.IsAny<short>(), It.IsAny<string>(), It.IsAny<long>()), Times.Never);
+        }
+
+        [Test]
+        [Category("UnitTest")]
+        public async Task Should_Trigger_Levy_Forecast()
+        {
+            // Arrange
+
+            // Act
+            await _sut.Handle(_event);
+
+            // Assert
+            _levyForecastServiceMock.Verify(mock => mock.TriggerLevyForecast(It.IsAny<short>(), It.IsAny<string>(), It.IsAny<long>()), Times.Once);
```

Comment:
Should match on specific known fixture values rather than It.IsAny<string>() (etc( to improve the test.

Reply:
Agree, changed